src/components/FullDisplay/FullDisplay.scss

```diff
@@ -1,9 +1,17 @@
 .full-display {
   &-container {
-    margin-top: 1rem;
     display: flex;
     flex-direction: row;
     gap: 0.5rem;
+    width: 520px;
```

Comment:
My be it's better to use relative value for responsive layout?

Reply:
Relative width will not correspond to the exact px value from the designs because of different root font size

Reply:
What if you use percentages?